sassoftware/vscode-sas-extension
# fix: externalized error message and toggle developer tools (#734)

**Summary**

Fix #734 

Suggestion from [jbodart-argenx](https://github.com/jbodart-argenx): https://github.com/sassoftware/vscode-sas-extension/issues/685#issuecomment-1849425397


1. Externalized error message 'There was an error executing the SAS Program. See [console log] for more details.'
2. The 'console log' in the show error message dialog should be a link to open the developer tools.

**Testing**
1. Change the language to zh-cn to check if the error message is change to Chinese.
2. Click the hyper link [console log] to see if the developer tools is opened.
